README.adoc

```diff
@@ -292,7 +301,7 @@ The `_configs/build-docs.yml` file is the brains of any LDCMF application.
 It defines the sequential compilation routine and ensures all assets are in place for the final artifact rendering operations.
 
 === LDCMF Build Strategies
-
+// tag::build-strategies[]
 LiquiDoc CMF isn't much without its ability to maintain strictly “DRY” single sourcing while still producing diverse output.
```

Comment:
`The strength of  LiquiDoc CMF is its ability to maintain strictly “DRY” single sourcing while still producing diverse output.`

Reply:
@briandominick did you see this one?